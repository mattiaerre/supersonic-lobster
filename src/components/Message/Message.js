import React, { PropTypes } from 'react';

const Message = ({ message, className }) => (<div className={`Message ${className}`}>{message}</div>);

Message.propTypes = {
  message: PropTypes.string.isRequired,
  className: PropTypes.string
};

Message.defaultProps = {
  className: ''
};

export default Message;
