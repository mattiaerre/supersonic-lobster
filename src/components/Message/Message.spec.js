import React from 'react';
import renderer from 'react-test-renderer';
import Message from './Message';

describe('<Message />', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Message message="Hello from Jest" />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
