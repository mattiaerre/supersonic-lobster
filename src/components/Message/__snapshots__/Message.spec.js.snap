exports[`<Message /> renders correctly 1`] = `
<div
  className="Message">
  Hello from Jest
</div>
`;
